src/makielayout/blocks/textbox.jl

```diff
@@ -210,6 +213,26 @@ function initialize_block!(tbox::Textbox)
 
     on(events(scene).keyboardbutton, priority = 60) do event
         if tbox.focused[]
+            ctrl_v = (Keyboard.left_control | Keyboard.right_control) & Keyboard.v
+            if ispressed(scene, ctrl_v)
+                local content::String = ""
+                local no_err::Bool = true
+                try
+                    content = clipboard()
+                finally
+                    for char in content
+                        if is_allowed(char, tbox.restriction[])
+                            insertchar!(char, cursorindex[] + 1)
+                        else
+                            no_err = false
```

Comment:
This doesn't really seem like an error case, so I'd rather call this `all_chars_allowed` or something like that.
Also, we should add a catch case and rethrow, no? Or at least warn...

Reply:
Excuse the clumsy implementation.

My thinking here was that it would be obvious that the copy/paste stopped.
I take your point, though, it would make more sense if I check for disallowed characters in the string first, and just abort the paste if any is found. Will make changes in the morning

Reply:
No worries, this is great :) Maybe just use:
```julia
if all(char-> is_allowed(char, tbox.restriction[]), content)
   foreach(char-> insertchar!(char, cursorindex[] + 1), content)
   return Consume(true)
else
    return Consume(true)
end
```

Reply:
Thanks! 

That second `Consume` returns `false` instead though, correct? But yes, I get the idea.